app/src/ui/app.tsx

```diff
@@ -2469,6 +2520,14 @@ export class App extends React.Component<IAppProps, IAppState> {
       kind: HistoryTabMode.History,
     })
   }
+
+  private inNoRepositoriesBlankSlateState: () => boolean = () => {
```

Comment:
First, 😍 , second: I don't believe we're passing this anywhere such that it needs to be bound

```suggestion
  private inNoRepositoriesBlankSlateState() {
```